lib/Sabberworm/CSS/Parser.php

```diff
@@ -516,13 +519,18 @@ private function peek($iLength = 1, $iOffset = 0) {
 
 	private function consume($mValue = 1) {
 		if (is_string($mValue)) {
+			$noLines = substr_count($mValue, "\n");
 			$iLength = $this->strlen($mValue);
 			if (!$this->streql($this->substr($this->iCurrentPosition, $iLength), $mValue)) {
 				throw new UnexpectedTokenException($mValue, $this->peek(max($iLength, 5)));
 			}
+			$this->iLineNum += $noLines;
 			$this->iCurrentPosition += $this->strlen($mValue);
 			return $mValue;
 		} else {
+			$substring = $this->substr($this->iCurrentPosition, $mValue);
```

Comment:
Why get the substring here when we already have it as `$sResult` two lines down?